projects/assets/src/translations/en/common.json

```diff
@@ -132,7 +132,8 @@
         "password": "Password entered is not valid.",
         "uid": "UID is not valid.",
         "code": "Code is not valid.",
-        "email": "Email is not valid."
+        "email": "Email is not valid.",
+        "loginId": "Email is not valid."
```

Comment:
why need to add a different key with same value?

Reply:
The API verificationToken's request body is like:
`{ "purpose": "LOGIN", "**loginId**": "customer.test@sap.com", "password": "Password123." }`
if the loginId is not valid, throw a global error message with subject loginId instad of emai